layouts/index.html

```diff
@@ -5,8 +5,17 @@
         <img class={{ .Site.Params.AvatarSize | default "size-m" }} src='{{ .Site.BaseURL }}{{ .Site.Params.AvatarURL }}' alt="{{ .Site.Params.AvatarAltText|default "avatar" }}" />
         {{ end }}
 	</div>
-    <h1>{{ .Site.Title }}</h1>
-    <h3>{{ .Site.Params.Description }}</h3>
+
+    {{ if isset .Site.Params "usemarkdownaboutdescription" }}
+        {{ $about := where .Site.Pages "Params.type" "about" }}
+        {{ range $about | first 1 }}
+            {{- partial "about-description" . -}}
+        {{ end }}
+    {{ else }}
+        <h1>{{ .Site.Title }}</h1>
+        <h3>{{ .Site.Params.Description }}</h3>
+    {{ end }}
```

Comment:
I think this doesn't yet work as intended. Try setting the `useMarkdownAboutDescription` parameter to `false`, hugo will still render the about page if one exists.

Something else I stumbled upon is the fact that the title and description are removed if I set said parameter. At least for me it feels more intuitive to simply render them if they're set but not make their rendering dependent on other parameters.

So my suggestion would be something like this:

```suggestion
    <h1>{{ .Site.Title }}</h1>
    {{ if isset .Site.Params.Description }}
        <h3>{{ .Site.Params.Description }}</h3>
    {{ end }}
    {{ if .Site.Params "usemarkdownaboutdescription" }}
        {{ $about := where .Site.Pages "Params.type" "about" }}
        {{ range $about | first 1 }}
            {{- partial "about-description" . -}}
        {{ end }}
    {{ else }}
```

Would that make sense for you as well? I'm quite certainly not aware of all implications or your use cases.

And, any chance we could make the parameter a bit easier/shorter? :sweat_smile:

Reply:
Thanks for feedback. I changed it according to your suggestions. Now the boolean expression should also work as expected.

Also renamed the flag parameter.

Reply:
Looks great, thank you!